decorator_in_action.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+require_relative './capitalize_name_decorator'
+require_relative './trim_name_decorator'
+
+person = Person.new(22, 'maximilianus')
+puts person.correct_name
```

Comment:
- [ ] You don't have a method called `correct_name` in Person class, so this would return errors. Additionally the Person class should inherit from the Nameable class:

  ```
  require_relative './nameable.rb'

  class Person < Nameable
    ...
  
    def initialize(age, name = 'Unknown', parent_permission: true)
      ...
      super()
    end
  
    def correct_name
      @name
    end
  ...
  ```